src/OSPSuite.Core/Domain/ParameterIdentifications/CategorialParameterIdentificationRunMode.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Assets;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility.Collections;
+
+namespace OSPSuite.Core.Domain.ParameterIdentifications
+{
+   public class CategorialParameterIdentificationRunMode : ParameterIdentificationRunMode
+   {
+      public bool AllTheSame { get; set; }
+      public CalculationMethodCache AllTheSameSelection { get; } = new CalculationMethodCache();
+      public Cache<string, CalculationMethodCache> CalculationMethodsCache { get; } = new Cache<string, CalculationMethodCache>(onMissingKey: x => null);
+
+      public CategorialParameterIdentificationRunMode() : base(Captions.ParameterIdentification.RunModes.Category, isSingleRun: false)
+      {
+      }
+
+      public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager cloneManager)
+      {
+         base.UpdatePropertiesFrom(source, cloneManager);
+         var sourceCategorialParameterIdentification = source as CategorialParameterIdentificationRunMode;
+         if (sourceCategorialParameterIdentification == null) return;
+         AllTheSame = sourceCategorialParameterIdentification.AllTheSame;
+         CalculationMethodsCache.Clear();
+         foreach (var keyValue in sourceCategorialParameterIdentification.CalculationMethodsCache.KeyValues)
+         {
+            CalculationMethodsCache[keyValue.Key] = keyValue.Value.Clone();
+         }
+      }
+
+      public IReadOnlyList<string> SelectedCategories
```

Comment:
I moved the file to it's own file so diff is going to be hard unfortunately. But the idea is that this method was not returuning the categories for the "AllTheSame" situation